### ISSUE TITLE
Denoise sequences where R1 and R2 are combined together

### ISSUE THREAD

**Issue by galud27** [benjjneb/dada2#408]
Hello!
I want to use dada2 to denoise fastq files pair end reads; however, I don't have forward and reverse readings separated  instead the readings are joined together. I was looking at the tutorial but it seems I need to input Forward and reverse reads. Do you have any suggestions on how to denoise them? 

Thanks!

**Comment by benjjneb** [comment]
It's recommended to obtain the unjoined reads and proceed from there.

You can try processing the joined reads (as if they are forward reads alone) but results are not guaranteed because of the different way quality scores are assigned in the merged region.

**Comment by galud27** [comment]
Hi  Benjamin!
Thanks for your response. 
I am now  using data generated  from single forward  reads. I am following the tutorial on Big Data for FWD. I would like to generate a data frame similar to what you get when using the mergerPairs function to get  denoised sequences and abundances. 
 I was trying to make a data.frame from dadaFs object by taking $ sequence and $abundance but I cant generate it. Is there anything in dada2 similar to the data frame output in  mergepairs  but for forward reads?

I am hoping to get all the denoised sequences and convert them into fasta files and run them in a different pipeline. I did see that there is an option to generate fasta files from the seqtab, but this is only the unique fasta and I would like to get all the fasta file after running sample inference. 

Thanks for your help!

**Comment by benjjneb** [comment]
The dada-class objects themselves have such a data.frame: `dadaFs[[1]]$clustering` (for sample 1) and so forth.

You can also just run `uniquesToFasta` on each dada-class output to get per-sample fasta.

**Comment by benjjneb** [comment]
Assuming that worked, but feel free to re-open if not.

**Comment by galud27** [comment]
Hi!
Thank you. 
Yes, I was using the uniquetoFasta, although I couldn't use it on the output from dada2 algorithm. I had to convert it into a sequence table and then I got a fasta file. 
However,using the uniquetoFasta gives me only the representative unique sequences, and I am hoping to get all the sequences generated after running dada2, but I dont think there's a function to extract all fasta files, right?

The dadaFs[[1]]$clustering gives me the data frame to convert it to fasta files, but I am trying to figure a way to make loop and run all my samples!

Thanks for your help!
